org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/dom/ASTConverter15JLS8Test.java

```diff
@@ -667,10 +668,8 @@ public void test0016() throws JavaModelException {
 		char[] source = sourceUnit.getSource().toCharArray();
 		assertTrue("Not a compilation unit", result.getNodeType() == ASTNode.COMPILATION_UNIT);
 		CompilationUnit compilationUnit = (CompilationUnit) result;
-		String expectedProblems =
-			"Pair is a raw type. References to generic type Pair<A,B> should be parameterized\n" +
-			"Pair is a raw type. References to generic type Pair<A,B> should be parameterized";
```

Comment:
what made these problems go away?

Reply:
The change in `org.eclipse.jdt.core.tests.dom.ConverterTestSetup.setUpJavaProject(String, String)` that added this line
`this.currentProject.setOption(JavaCore.COMPILER_PB_RAW_TYPE_REFERENCE, JavaCore.IGNORE);`
, otherwise lot of converter tests (104 to be exact) failed with unexpected "raw type" warnings.